lib/modules/datasource/adoptium-java/index.ts

```diff
@@ -29,7 +30,7 @@ export class AdoptiumJavaDatasource extends Datasource {
     url: string,
     page: number
   ): Promise<Release[] | null> {
-    const pgUrl = `${url}&page=${page}`;
+    const pgUrl = joinUrlParts(url, `&page=${page}`);
```

Comment:
are you sure this works as expected for query parts?

Reply:
Yes: https://github.com/jfromaniello/url-join/blob/e8316b706d6eb10c81e661e8595e5386a5147de3/test/tests.js#L5-L8

Reply:
Yes: https://github.com/jfromaniello/url-join/blob/e8316b706d6eb10c81e661e8595e5386a5147de3/test/tests.js#L76-L85

Reply:
didn't know that. nice 👍